EventsExpress.Core/Services/EventService.cs

```diff
@@ -111,6 +121,27 @@ public async Task<OperationResult> DeleteUserFromEvent(Guid userId, Guid eventId
             return new OperationResult(false, "Visitor not found!", "visitorId");
         }
 
+        public async Task<OperationResult> PromoteToOwner(Guid userId, Guid eventId)
```

Comment:
Let's extract these methods to separate service. This service is overloaded with responsibility.